### PR TITLE
#9 update method docs

### PR DESCRIPTION
**Pull request by applehit16**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
 /**
- * get offset of element
- * @param {dom element} el 
+ * @typedef {Object} Offset
+ * @property {number} top
+ * @property {number} left
```
Comment (ThomasKranitsas) [inline js/main.js]: Can we remove the above lines?

There seems to be documentation without any method?

Comment (applehit16) [reply js/main.js]: @ThomasKranitsas it is a `typedef` for type `Offset`, but I can make it inline

Comment (ThomasKranitsas) [reply js/main.js]: thank you!

Comment (applehit16) [reply js/main.js]: @ThomasKranitsas updated!

**Pull request merged**